{{ cookiecutter.name }}/.github/actions/build/action.yml

```
@@ -0,0 +1,42 @@
+name: build
+
+runs:
+  using: composite
+
+  steps:
+    - name: load cached poetry installation
+      id: cached-poetry
+      uses: actions/cache@v3
```

```suggestion
      uses: actions/cache@v4
```

Тут кажется по всему ПР пора обновить версии, чтобы работали с node20, а то node16 начали депрекейтить